frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx

```diff
@@ -75,6 +75,15 @@ const InstalledHintBlock: React.FC<OperatorHubItemDetailsHintBlockProps> = ({
     isList: false,
     namespaced: true,
   });
+
+  if (!subscription?.status?.installedCSV) {
+    return (
+      <HintBlock className="co-catalog-page__hint" title="Operator Not Yet Installed">
```

Comment:
Hey, Kim. We might look at the messaging here. It's odd to see an "Uninstall" button on the same page with "Operator Not Yet Installed." Is the operator is still installing or installation failed? Is there something in the Subscription resource that will let us give a more specific status to the user?

cc @tlwu2013

Reply:
I didn't see anything in the subscription resource that would indicate that the CSV hasn't been found vs a failure, but I can look again.

I would love some suggestions on what the UI should respond with installedCSV is undefined or missing.  I just went with what was suggested in the bug.

Reply:
Looking below, I see we try to fallback to the Subscription link when there's no CSV. I think that's the intended behavior, and I'd do that for this case as well.

Reply:
Here is where this bug gets kind of weird.  On line 71, installedCSV is created by passing information. 
```
 const [installedCSV] = useK8sWatchResource<ClusterServiceVersionKind>({
    kind: referenceForModel(ClusterServiceVersionModel),
    name: subscription?.status?.installedCSV,
    namespace: subscription?.metadata?.namespace,
    isList: false,
    namespaced: true,
  });
```
In the case described in the bug, subscription.staus.insatlledCSV is actually undefined, but a item is returned from useK8sWatchResource.  So the logic used below on line 88:
```
 const to = installedCSV
    ? `${nsPath}/clusterserviceversions/${installedCSV?.metadata?.name ?? ''}`
    : `${nsPath}/subscriptions/${subscription.metadata.name ?? ''}`;
```

Would always return true.  What is actually happening is that the results from the useK8sWatchResource appears to return information on another installed operator where there was a valid CSV.  

The only piece of data that I could find to indicate this situation where need information is known except a valid CSV is checking for `subscription?.status?.installedCSV`.

Reply:
Oh weird. I'm seeing this too. We probably need to beef that logic up a bit more.

Reply:
I think Sam is right and https://github.com/openshift/console/pull/9346 does cover this case (since it overrides the install hint block), but I'll double check.

Reply:
Thanks @rebeccaalpert for taking a look.

Reply:
Hi @kdoberst! I think I might have written that, good catch. We probably need to add a condition there to make sure that nothing gets passed into `useK8sWatchResource` if `subscription.status.installedCSV` is empty. Something like:
```tsx
 const [installedCSV] = useK8sWatchResource<ClusterServiceVersionKind>( subscription?.status?.installedCSV ? {
    kind: referenceForModel(ClusterServiceVersionModel),
    name: subscription?.status?.installedCSV,
    namespace: subscription?.metadata?.namespace,
    isList: false,
    namespaced: true,
  } : null);
 ```

Reply:
@kdoberst - I tried the subscription YAML update in https://bugzilla.redhat.com/show_bug.cgi?id=1970466 against my PR and it doesn't trigger the new "isInstalling" hint block from what I can tell. I also wasn't seeing the error though. Did you use a different YAML to test it?